### PR TITLE
Fixes some misspellings, typos, grammar, and punctuation

### PR DESCRIPTION
**Pull request by colindean**
"PubSab" bothered me. I put a little more effort into cleaning up some of the rest, too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,22 +2,20 @@
...
-3. _Stay Platform and Technology Independent_ - The abstraction layer over infrastructure should be independent from Application
-technological stack and all kind of data transfer should be packaged as an Event for safe distribution, transfer, broadcasting.
+3. _Stay Platform and Technology Independent_ - The abstraction layer over infrastructure should be independent from application technology stack and all kind of data transfer should be packaged as an event for safe distribution, transfer, broadcasting.
 
 # Building
-For building from source you will need [Rust Language](https://rust-lang.org) installed. There is only one command for building this project
-on all platforms which is supported by Rust.
+For building from source you will need [Rust language](https://rust-lang.org) installed. There is only one command for building this project on all platforms which is supported by Rust.
+
+This project mainly tested on Linux, BSD, Windows, macOS and Android (experimental).
```
Comment (emintham) [inline README.md:17]: This project *is* mainly tested, perhaps?

File: README.md [hunk 2]
```diff
@@ -32,8 +30,8 @@ This project mainly tested on Linux, BSD, Windows, MAC and Android(experimental)
 # Roadmap for release
 - [x] Distributed Tree/Graph structure with automatic lookup
 - [x] Event path calculation between Tree/Graph nodes
-- [x] API Client subscriptions for each node and Event delivery
-- [x] Event broadcasting and round rubin load balancing using statefull path calculation
+- [x] API client subscriptions for each node and event delivery
+- [x] Event broadcasting and round rubin load balancing using stateful path calculation
```
Comment (emintham) [inline README.md:34]: round robin?

**Comment by tigranbs** [comment]
@colindean thanks a lot for fixing this!

File: README.md (hunk 2 above)
Comment (tigranbs) [reply README.md:34]: @emintham you are correct it should be `round robin`

**Pull request merged**